### PR TITLE
feat(InteractionResponse): createdTimestamp

### LINKED ISSUE #8916: Add `createdTimestamp` property to `InteractionResponse`
### Which package is the feature request for?

discord.js

### Feature

As title. Since we do have an ID, the `createdTimestamp` property can be provided. Although the user can deconstruct the ID themselves, there is no reason to not have this property provided.

### Ideal solution or implementation

The property is added

### Alternative solutions or implementations

_No response_

### Other context

_No response_

### PR DESCRIPTION
**Pull request by BenjammingKirby**
**Please describe the changes this PR makes and why it should be merged:**
InteractionResponses have IDs, we can just destructure it and get createdAt and createdTimestamp, (it's a getter because other such properties on other classes were getters)
Close #8916 
**Status and versioning classification:**
- Code changes have been tested against the Discord API, or there are no code changes
- I know how to update typings and have done so, or typings don't need updating
- This PR changes the library's interface (methods or parameters added)

### CODE REVIEW

File: packages/discord.js/src/structures/InteractionResponse.js [hunk 1]
```diff
@@ -21,6 +22,24 @@ class InteractionResponse {
     this.client = interaction.client;
   }
 
+  /**
+   * The timestamp the interaction's response was created at
```
Comment (MrMythicalYT) [inline packages/discord.js/src/structures/InteractionResponse.js:26]:
```suggestion
   * The timestamp the interaction response was created at
```

File: packages/discord.js/src/structures/InteractionResponse.js [hunk 2]
```diff
@@ -21,6 +22,24 @@ class InteractionResponse {
...
+   * @type {number}
+   * @readonly
+   */
+  get createdTimestamp() {
+    return DiscordSnowflake.timestampFrom(this.id);
+  }
+
+  /**
+   * The time the interaction's response was created at
```
Comment (MrMythicalYT) [inline packages/discord.js/src/structures/InteractionResponse.js:35]:
```suggestion
   * The time the interaction response was created at
```

**Review by MrMythicalYT — CHANGES_REQUESTED** [review]

**Comment by MaksiRose** [comment]
As far as I know, `createdTimestamp` is normally a normal property rather than a method. Is there a reason this is different here?

**Comment by imranbarbhuiya** [comment]
This isn't a method but a getter that behaves like a property but u can return result dynamically

**Comment by imranbarbhuiya** [comment]
using property will store the `Date` and `number` in memory and using getter, it'll generate when needed without storing. Ig this is the reason for not using property but not sure tho

**Comment by Qjuh** [comment]
> using property will store the `Date` and `number` in memory and using getter, it'll generate when needed without storing. Ig this is the reason for not using property but not sure tho

That, but also that it doesn’t have to be calculated at all if it isn’t used.

**Review by kyranet — APPROVED** [review]

**Review by SpaceEEC — APPROVED** [review]

**Review by vladfrangu — APPROVED** [review]

**Pull request merged**